modules/virt-configuring-runstrategy-vm.adoc

```diff
@@ -8,11 +8,6 @@
 
 You can configure a run strategy for a virtual machine (VM) by using the command line.
 
-[IMPORTANT]
-====
-The `spec.runStrategy` and `spec.running` keys are mutually exclusive. A VM configuration that contains values for both keys is invalid.
-====
-
```

Comment:
Instead of removing it, I would mention that using `spec.running` is not recommended since its deprecated in favor of using `runStrategy`

Reply:
After offline discussion we decided to remove this section